settings/prod_settings.py

```diff
@@ -40,6 +40,6 @@
 print(f"[BACKEND] Using {DB_TYPE} database: {os.environ.get('DATABASE_NAME')}")
 
 
-ALLOWED_HOSTS = [os.environ.get("URL")]
+ALLOWED_HOSTS: list[str | None] = [os.environ.get("URL")]
```

Comment:
Ideally this should just be list[str] and never allow None

Reply:
Yes but while **local** is already defined and set in stone, **prod** still have a possibility of been `None`. Both of them need to set with the same type otherwise `mypy` is throwing an error so we can leave it with `list[str | None]` or set `list[str]` and use `# type: ignore`

Reply:
Ah I see. I guess for now we can keep it as list[str | None] like you suggested then.